src/test/java/com/wikia/webdriver/testcases/desktop/spamwikireview/ViewSpamWikisTests.java

```diff
@@ -0,0 +1,106 @@
+package com.wikia.webdriver.testcases.desktop.spamwikireview;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.communities.desktop.pages.SpamWikiReviewPage;
+
+import org.apache.commons.lang3.EnumUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+@Test(groups = "SpamWikiReview")
+public class ViewSpamWikisTests extends NewTestTemplate {
+
+
+  private final String XPATH_LANGUAGE_COLUMN = "./td[4]";
+  private final String QUERY_PARAM_LANG = "?lang=";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this field "QUERY_PARAM_LANG" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)